### DISCUSSION TITLE
Are we just reinventing folders?

### DISCUSSION THREAD

**Discussion by alevangel** [TfTHacker/DashboardPlusPlus#35] [Q&A]
Thanks for sharing this work!

I’ve set it up and started using it, but I still have some questions. Specifically, I’m trying to understand how this method actually differs from a traditional folder and file system.

Each "Dashboard" seems to function like a regular folder but is created manually. Is that correct?

Also, in the sample vault shared here, the file [What is Spaced Repetition and How to Use It.md](https://github.com/TfTHacker/DashboardPlusPlus/blob/master/Research/What%20is%20Spaced%20Repetition%20and%20How%20to%20use%20it.md) isn’t displayed on any dashboard. In cases like that, how do you retrieve or keep track of files over time?

**Comment by TfTHacker** [comment]
This is a good question. And as with everything, it comes down to personal preference. The thing I find with folders is it's easy to bury information and not rediscover it. Or there might be information that you want to store but isn't relevant to your everyday needs. The idea with this dashboard is that you end up creating an optimized index. 

I find that the additional effort helps me retain better what information I've stored. Consider at a second level of improving retention about your notes. Where this system pays off in the future is when you come back to a dash dashboard. It helps you to work you the work you the past where folders where the file, where the file, you have to really, you have to understand what's going on. Each approach, each advantage. 

Having said all that, I like to keep things very simple. There are a lot more fancy things that we could do with this dashboard, but I like the fact that it's just a simple markdown list with indentation. It's easy to maintain and it's easy to add to. In fact, other than the CSS snippets, you don't even need plugins if you don't want.

**Reply by alevangel** [reply]
Thank you for your reply, you have clarified some interesting aspects. However, I still have one concern: what happens when the system starts to grow and some files are ‘orphaned’ outside the dashboards? I imagine it can become complex to retrieve them or even remember that they exist if we don't use them for a while.

Do you have any strategy for keeping track of even the less relevant files or those that do not immediately belong to a specific dashboard?

**Reply by TfTHacker** [reply]
I don't index all files in the dashboard. It really depends on the type of notes and their long-term use. For example, I don't add meeting notes to the dashboard since those are more for history and are accessed via search if needed.

In some cases, I am using Dataview for queries on a page or on a dashboard. 

I take into consideration doing lots of linking well, which helps with backlink functionality (see my plugin Strange New Worlds) and also the graph view in some cases.
